rs/types/types/src/canister_log.rs

```diff
@@ -16,23 +16,20 @@ pub const MAX_AGGREGATE_LOG_MEMORY_LIMIT: usize = 4 * KiB;
 pub const DEFAULT_AGGREGATE_LOG_MEMORY_LIMIT: usize = 4 * KiB;
 
 /// The maximum size of a delta (per message) canister log buffer.
-pub const MAX_DELTA_LOG_MEMORY_LIMIT: usize = 4 * KiB;
+pub const MAX_DELTA_LOG_MEMORY_LIMIT: usize = 100 * KiB;
```

Comment:
FYI: It's ok to increase this value, because the delta log size is calculated as minimum of the current log size (which is 4 KiB at the moment) and the maximum value, so it's a no-op for prod and stays 4 KiB.